room/views.py

```diff
@@ -69,7 +69,7 @@ def invite(request):
                 if not invitations:
                     # erm, kind of random but just want to get it working
```

Comment:
I recommend that comments are written as sentences in good English. They should start with a capital letter and (importantly) finish with a full-stop (except when they act as titles, in which case title-case should be used.

Reply:
Lol that one escaped me. I was talking to myself and forgot to remove it. Will delete.